docs/guide/interactions/checks.rst

```diff
@@ -187,9 +187,51 @@ As an example, here is a command which only runs if a user has a certain role, `
     async def check_example(interaction: discord.Interaction):
         await interaction.send_message('Checks passed!', ephemeral=True)
 
+Global Interaction Checks
+-------------------------
+
+In addition to adding individual or combined checks onto specific app commands or context menus, it is also possible to create global interaction checks. For these, any interaction
+passed through the bot for either app commands or context menus will first run this check.
```

Comment:
i'll have to double check this later to see if it works on context menus or not, but it should work on all slash commands

Reply:
I did verify this one already, it works on both context menus and slash commands.

Reply:
then that section should be fine